src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -262,6 +265,7 @@ def _resampling_request_sender(self) -> Sender[ComponentMetricRequest]:
                 config=self._resampler_config,
             )
             self._resampling_actor = _ActorInfo(actor, channel)
+            asyncio.create_task(self._resampling_actor.actor.start())
```

Comment:
This shouldn't be needed, as it is started in `_start()`.

Reply:
No, this is the whole point of this PR, it's not started in `_start()` if you call `_start()` _before_ you call any function that uses `_data_sourcing_request_sender()` as the actor has not been created at that point.

So if you don't call `logical_meter()` or any similar method that indirectly uses this, it will never be started.

Reply:
Oh, the thing here is you are talking about a case where `microgrid.initialize()` isn't being called, right? That's the case in the new test at least.

Reply:
No no, it's even independent of that. `microgrid.initialize()` just calls `_start()`. Start will start datasoucring and resampling actors if they are set at that point.

They are set only on demand when requested however. So if you only do `.logical_meter()` after `initialize()` you will have the same problem.